src/librustc_span/symbol.rs

```diff
@@ -112,10 +112,13 @@ symbols! {
     //
     // As well as the symbols listed, there are symbols for the the strings
     // "0", "1", ..., "9", which are accessible via `sym::integer`.
+    //
+    // Keep this list in sorted order, as defined by the Unix `sort` utility.
```

Comment:
Apparently this order is different than what the "Sort Lines" command in VSCode does :(
I tried to sort this list before and immediately found differences between me and the previous person who sorted the list using some different editor or sorting utility.

Reply:
Well now the correct ordering is documented! :)

Reply:
How do I run sort on this list? copy the list elsewhere, sort it there and copy it back?

Reply:
In vim you can run a section of selected text through a shell command using the `!` operator. That's how I do it.

Perhaps we should make the proc macro code check the ordering, to guarantee that everybody agrees on the correct order. Another thing for a follow-up.

Reply:
I have implemented order checking in #74203.